### PR TITLE
Update how-to-guide headings

### PR DESCRIPTION
**Pull request by sean1588**
fixes: https://github.com/pulumi/registry/issues/1823

This PR moves to always using the title as the heading for the how to guide pages so that they always match the breadcrumb, since that is the same way the breadcrumb is being populated. Prior to this we were using the h1 property in the front matter if it existed and would fall back to using the title if it wasn't present. I have switched it to always use the title so we always match the breadcrumb.

_I added in the how to guides for aws so we can test it out. I'll remove the commit with these additional files once we approve the way it looks._

image

### CODE REVIEW

**Review by susanev — COMMENTED** [review]
headings look good, but i think the content on the how-to guides is possibly being constrained too much, at least with the heading

Screen Shot 2022-12-15 at 5 43 25 PM
Screen Shot 2022-12-15 at 5 42 38 PM

**Comment by anita-trimbur** [comment]
im seeing the same content width issue as susan at other breakpoints in the how-to-guides also.
Screen Shot 2022-12-16 at 12 38 15 PM

File: themes/default/layouts/registry/how-to-guide.html [hunk 1]
```diff
@@ -1,11 +1,12 @@
 {{ define "main" }}
+{{ $lastUrlSlug := index (last 1 (split (delimit (split $.Page.File "/") "," "") ",")) 0 }}
```
Comment (kimberleyamackenzie) [inline themes/default/layouts/registry/how-to-guide.html:2]: Is this needed? or maybe a rebase leftover?

**Review by kimberleyamackenzie — APPROVED** [review]
1 comment on the code, but LGTM!

File: themes/default/layouts/registry/how-to-guide.html (hunk 1 above)
Comment (sean1588) [reply themes/default/layouts/registry/how-to-guide.html:2]: yes thank you so much. will remove!

**Review by anita-trimbur — APPROVED** [review]

**Pull request merged**